### ISSUE TITLE
False positive with UselessVariable

### ISSUE THREAD

**Issue by Chi-teck** [slevomat/coding-standard#1527]
Example:
```php
public function example() {
  $this->storage->save($entity = $this->storage->create());
  return $entity;
}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/d5766ddbf691c4288c54c0d71542396d61b5b490
